src/components/ModelSteps/Scan.jsx

```diff
@@ -49,36 +49,38 @@ const Scan = ({ currentStep }) => {
     setIsImportFileModalOpen(true)
   }
 
-  const onFileChange = file => {
+  const onChangeFile = useCallback(file => {
     if (file) {
       setFile(file)
       setIsImportFileModalOpen(false)
     }
-  }
+  }, [])
```

Comment:
I don't see the point of using useCallback here. The operation of setting is costless.

Reply:
Because this function is also used in the useEffect, I did so to avoid repeating the 2 setters